docs/howto/openstack/nova/resize-server.md

```diff
@@ -0,0 +1,125 @@
+# Resizing a server
+
+This guide will walk you through the required steps to change the number of CPU cores and the amount of memory your server has access to, this is done my changing the server's [flavor](/reference/flavors/).
+
+[Resize](https://docs.openstack.org/nova/latest/admin/configuration/resize.html) (or Server resize) is the ability to change the flavor of a server, thus allowing it to upscale or downscale according to user needs. A resize operation is a two-step process for the user: the first step is to perform the resize, and the second step is to either confirm (verify) success and release the old server or to declare a revert to release the new server and restart the old one.
+
+## Prerequisites
+
+You need to have a server you wish to resize. Additionally, if you prefer to work with the OpenStack CLI, then make sure to properly [enable it first](/howto/getting-started/enable-openstack-cli/).
+
+## Listing available flavors
+
+=== "{{gui}}"
+    Navigate to the server list.
+
+    ![navigation-panel](assets/resize-server/01-left-side-panel.png)
+
+    Find the server you want to resize in the list and on the right hand side click on it's menu button.
+    
+    ![menu-button](assets/resize-server/02-menu-button.png)
+
+    Click on _Modify Server_.
+
+    ![server-menu](assets/resize-server/03-menu-list.png)
+
+    On this panel near the top, find the section called _Flavor_, this is the current flavor used by the server. Press on the dropdown menu to see all available flavors.
+
+    ![server-edit](assets/resize-server/04-modify-server.png)
+=== "OpenStack CLI"
+    To list all available flavors you can simply run `openstack flavor list`, but that will return a very long unsorted list, instead we recommend the following command:
+
+    ```bash
+    openstack flavor list -c Name -f value | grep '[0-9]c[0-9]' | sort -V
+    ```
+
+    The printout is a simple and clean list, sorted by the compute type, the number of cores and then by the amount of memory.
+
+    ```plain
+    X.1c1gb
+    X.1c2gb
+    X.1c4gb
+    X.2c2gb
+    X.2c4gb
+    X.2c8gb
+    X.2c16gb
+    X.4c4gb
```

Comment:
Rather than the `X` prefix, you might as well use `B` which constitutes the vast majority of our current flavors.

Reply:
I thought it would be best to use the same naming convention as the official flavor reference document, no? https://docs.cleura.cloud/reference/flavors/#naming-convention

I could skip the fake printout and simply describe that the list will be grouped by compute type and sorted by cores and then ram and printed in the X.YcZgb form. If you prefer that instead.

Reply:
The reference is a reference. This how-to is a how-to. :)

I think the printout is fine, but it shouldn't use a prefix that doesn't actually exist in any flavor. Hence the suggestion to use `b.` and not `X.`.